### DISCUSSION TITLE
Do we have plan to support SSE (Server Sent Event) and Websocket?

### DISCUSSION THREAD

**Discussion by Lan-Phan** [illacloud/illa-builder#963] [General]
I think SSE and Websocket is somehow similiar with Rest api (of course it's more complicated), do we have plan to support it? Once we have it, content on frontend will be more dynamic.

Thanks.

**Comment by wenzhuo2022** [comment]
Hello, we plan to release a new feature that can support similar content in this year. But I'm not sure whether it will meet your needs. Could you please describe your usage scenarios, such as how to use SSE and Websocket, where to use it, etc? And we will evaluate our product proposal based on your feedback.
